### ISSUE TITLE
MoltenVK implementation

### ISSUE THREAD

**Issue by official-antique** [emuPlace/emuThreeDS#1]
Once this is done Citra will finally work on Apple devices.

**Comment by Ayoendit** [comment]
I cant wait, i will love it.
its been a long time ive been waiting for. You can do it man, i believe in you!

**Comment by official-antique** [comment]
@Ayoendit it's actually done now but I'll need to make a new issue related to the file system which appears to be the next major issue.

**Comment by official-antique** [comment]
Fixed MoltenVK as far as crashes are concerned, issue may be reopened if more rendering issues appear later.
